### DISCUSSION TITLE
ArchR multiome scATAC part cell type definition

### DISCUSSION THREAD

**Discussion by Jiahuan90** [GreenleafLab/ArchR#1462] [Questions / Documentation] [answered]
Hi rcorces, 

In the multiome data, I have define the cell type in scRNA-seq part and import cell type information to the ArchR project. In the ArchR project, I also to LSI dim reduction and UMAP embedding. However, I meet a problem on cell type definition.

For example, in LSI ATAC, cluster 1 have 300 T cells and 200 B cells, which T cells and B cells definition are from scRNA-seq. So how could I define the cluster1 without conflict to the scRNA-seq part? 

Do you have any suggestions?

Thank you for your help~

**Comment by rcorces** [comment, marked as answer]
Sorry - We do not have bandwidth to answer questions that are specific to one individual end-user's analysis.

**Reply by Jiahuan90** [reply]
Thank you for your prompt resonse, I have figured out the problem:)
